.github/workflows/scheduled-snapshots.yml

```diff
@@ -132,3 +135,41 @@ jobs:
           --head "schutzbot:snapshots-${{ steps.generate_suffix.outputs.suffix }}"
         popd
 
+    - name: Clone osbuild repository
+      uses: actions/checkout@v2.3.4
+      with:
+        repository: osbuild/osbuild
+        path: osbuild
+        fetch-depth: 0
+
+    - name: Update schutzfile and open PR in osbuild repository
+      env:
+        GITHUB_TOKEN: ${{ secrets.SCHUTZBOT_GH_TOKEN }}
+        SUFFIX: ${{ steps.generate_suffix.outputs.suffix }}
+      run: |
+        pushd osbuild
```

Comment:
I think that this script is the same one as above. What about extracting it to a new file and reusing it? Would it be viable?

Reply:
> I think that this script is the same one as above. What about extracting it to a new file and reusing it? Would it be viable?

Yes, thanks for the suggestion, I've put the whole thing inside a script.